Tools/MonoGame.Content.Builder.Editor.Launcher.Bootstrap/MonoGame.Content.Builder.Editor.Launcher.Bootstrap.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <Description>The MonoGame Framework Content Builder Editor (MGCB-Editor) is a graphical tool used for editing your content projects ready for processing.</Description>
+    <BaseOutputPath>..\..\Artifacts\MonoGame.Content.Builder.Editor.Launcher\Bootstrap</BaseOutputPath>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>mgcb-editor</ToolCommandName>
+    <PackageId>dotnet-mgcb-editor</PackageId>
+    <AppendTargetFrameworkToOutputPath>False</AppendTargetFrameworkToOutputPath>
+    <AssemblyName>mgcb-editor</AssemblyName>
+    <ImplicitUsings>true</ImplicitUsings>
```

Comment:
Shouldn't we add package references for all 3 editors in the event users would try to install ```dotnet-mgcb-editor``` from the command line? This would automatically install the implementations as well (maybe we could also remove the implementations from the ```dotnet-tools.json``` this way?).

Reply:
Unfortunately it does not work that way :/

All that would happen if we did that would be that the dlls of the referenced project would also get packaged in.